test/block_proxy/component.adb

```diff
@@ -73,6 +106,20 @@ package body Component is
          and then D'Length = Block_Size (Server) * Block_Server.Length (Cache (I).S)
       then
          Block_Server.Write (Server, Cache (I).S, D);
+         pragma Warnings (Off, "*never be executed*");
```

Comment:
I wonder whether this block would get less ugly if we made `Print_Content` a parameter of `Print_Buffer` which is called unconditionally. The subprogram could then return based on the value of `Print_Content` and all warning suppression could happen there.

Reply:
I fixed it in https://github.com/Componolit/gneiss/pull/91/commits/b98e2037037b23a2ed8b1c34cad118d760341a13, there are no warnings left to be suppressed.

Reply:
The code is much gneisser now! ;)